extensions/amp-analytics/amp-analytics.md

```diff
@@ -173,6 +173,87 @@ In this example, we specify the `config` attribute to load the configuration dat
 <amp-analytics config="https://example.com/analytics.account.config.json">
 ```
 
+#### Configuration Rewriter
+
+The configuration rewriter feature is designed to allow analytics providers to dynamically rewrite a provided configuration. This is similar to the remote configuration feature but additionally includes any user-provided configuration in the request made to the sever. This currently can only be enabled by an analytics vendor.
+
+An analytics vendor will specify a `configRewriter` property with a server url.
```

Comment:
Try to avoid future tense:
"An analytics vendor will specify a `configRewriter` property with a server url."

to

"An analytics vendor specifies a `configRewriter` property with a server url."

Reply:
Done.